data/json/vehicleparts/vehicle_parts.json

```diff
@@ -639,7 +639,7 @@
     "looks_like": "minifridge",
     "damage_modifier": 80,
     "durability": 100,
-    "epower": "-45 W",
+    "epower": "-7 W",
```

Comment:
I can't find numbers for this quite so easily, but I would not expect a reduction like this given the minifridge results.